bims/templates/map.html

```diff
@@ -64,24 +65,114 @@
         </div>
     </script>
 
-    <script type='text/html' id='side-panel-template'>
-        <div class="right-panel">
-            <div class="row side-panel side-panel-info">
-                <div class="col-lg-12" id="content-panel">
+    {% if is_healthyrivers %}
```

Comment:
move all this into healthyrivers-map.html, use the extend method. No need to do condition, because the "side-panel-template" id will be replaced by healthtyrivers-map.html.